### PR TITLE
Some minor updates

### PR DESCRIPTION
**Pull request by jrappen**
See commits for details.

### CODE REVIEW

**Comment by jrappen** [comment]
Preview of changes to fancyhdr:

image

Preview of changes by adding `\pagecolor` to the latex template:

* enhanced readability with grey background
* emphasized code blocks

image

Fixed error message where `InspiredGithub` should've been `InspiredGitHub`, no typos in source code itself:

image

**Comment by jrappen** [comment]
@lise-henry FYI, when testing the changes of the output, crowbook doesn't use the current templates but the templates included in the version of crowbook used. Not sure whether you'd like to change that in `guide.book`. I was confused for a second, why my output style didn't change.

**Comment by jrappen** [comment]
This here seems test related for Travis. Might break after the next release?

image

**Comment by lise-henry** [comment]
Not sure what you mean about the templates ? If you specify a file in the configuration it should use this one, else it is a bug. If you don't you have to recompile, if you change the source file because it's included in the binary so this is normal.

Don't worry about travis I see where the problem is, i'll fix it when I merge. It will  probably be in a few days, though, because I think the LaTeX/Crowbook integration needs to be tweaked a little (e.g. \chaptermark will cause an error if the class doesn't have chapters so it needs to be detected, and I'd like to make headers, footers and papercolor a bit more configurable).

Anyway thanks for the PR, i'll merge it asap :)

**Comment by jrappen** [comment]
Maybe it's unclear what I meant. Let me try again.

I assumed the `guide.book` would be using the current templates. But actually the template files aren't referenced, so it's using the template files from the crowbook binary.

Not an issue in any case. Maybe worth adding to the docs as an FYI, though.

**Comment by jrappen** [comment]
I'd like to close this for now. Running into the following issue:

```text
🐦 CROWBOOK 📚 0.15.0
21:15:09 [DEBUG] (1) crowbook::book: Parsing chapter: ***...
21:15:09 [DEBUG] (1) crowbook::book: Parsing chapter: ***...
21:15:09 [DEBUG] (1) crowbook::book: Attempting to generate pdf...
21:15:09 [DEBUG] (1) crowbook::latex: Attempting to run LaTeX on generated file
21:15:11 [DEBUG] (1) crowbook::zipper: xelatex didn't return succesfully: This is XeTeX, Version 3.14159265-2.6-0.999991 (TeX Live 2019) (preloaded format=xelatex)
 restricted \write18 enabled.
entering extended mode
(./result.tex
LaTeX2e <2019-10-01> patch level 3
(/usr/local/texlive/2019/texmf-dist/tex/latex/base/book.cls
Document Class: book 2019/10/25 v1.4k Standard LaTeX document class
(/usr/local/texlive/2019/texmf-dist/tex/latex/base/bk10.clo))
(/usr/local/texlive/2019/texmf-dist/tex/latex/fontspec/fontspec.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/l3packages/xparse/xparse.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/l3kernel/expl3.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/l3kernel/expl3-code.tex
(/usr/local/texlive/2019/texmf-dist/tex/latex/l3kernel/l3deprecation.def))
(/usr/local/texlive/2019/texmf-dist/tex/latex/l3backend/l3backend-xdvipdfmx.def
))) (/usr/local/texlive/2019/texmf-dist/tex/latex/fontspec/fontspec-xetex.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/base/fontenc.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/base/tuenc.def))
(/usr/local/texlive/2019/texmf-dist/tex/latex/fontspec/fontspec.cfg)))
(/usr/local/texlive/2019/texmf-dist/tex/xelatex/xunicode/xunicode.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/tipa/t3enc.def)
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics/graphicx.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics/keyval.sty)
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics/graphics.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics/trig.sty)
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics-def/xetex.def))))
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel/babel.sty
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel/switch.def)
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel-german/ngerman.ldf
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel-german/ngermanb.ldf
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel/babel.def
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel/xebabel.def
(/usr/local/texlive/2019/texmf-dist/tex/generic/babel/txtbabel.def))))))
(/usr/local/texlive/2019/texmf-dist/tex/latex/fancyhdr/fancyhdr.sty)
(/usr/local/texlive/2019/texmf-dist/tex/latex/hyphenat/hyphenat.sty

Package hyphenat Warning: *******************************
(hyphenat)                * You have used the htt option.
(hyphenat)                * You are likely to get many Font Warning messages.
(hyphenat)                * These can usually be ignored.
(hyphenat)                *******************************.

) (/usr/local/texlive/2019/texmf-dist/tex/latex/xcolor/xcolor.sty
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics-cfg/color.cfg)
(/usr/local/texlive/2019/texmf-dist/tex/latex/graphics/dvipsnam.def)
(/usr/local/texlive/2019/texmf-dist/tex/latex/xcolor/svgnam.def))
Runaway argument?
 backref=page, bookmarks=true, bookmarksdepth=subsection, bookmarksnu\ETC.
! Paragraph ended before \@fileswith@ptions was complete.
<to be read again>
                   \par
l.49

?
! Emergency stop.
<to be read again>
                   \par
l.49

No pages of output.
Transcript written on result.log.

💣 ERROR Error during temporary files editing: xelatex didn't return succesfully
```

with the suggested changes of the latex template. Cannot reproduce every time and every repository where I am using it.

Where can I find the `result.log` mentioned?

**Comment by jrappen** [comment]
Looking at:

https://github.com/lise-henry/crowbook/pull/68/commits/b7f7e7ec3438987d48a711e686b23e8e64a23134

the only possible reason might be that it breaks without a `subtitle`? That's my first guess. Don't have time today to further test. Will report back.

Sorry for the mess. Should've tested this more thoroughly.
